tests/app/Jobs/NotifyUserOfNewGistCommentTest.php

```diff
@@ -66,4 +67,17 @@ public function itSendsEditCommentEmailWhenACommentHasBeenEdited()
 
         $this->assertEquals('emails.edit-comment', $this->mailSendData[0]);
     }
+
+    /**
+     * @test
+     */
+    public function itGeneratesUnsubscribeUrl()
+    {
+        $user = new User([
+            'github_id' => 987654,
+            'token' => 'ABC123',
+        ]);
+
+        $this->assertEquals('https://giscus.co/unsubscribe?id=987654&hash=2e8f94ecbd8dba1da15c888b2ef0dbd3', $user->getUnsubscribeUrl());
```

Comment:
I think you would need to assert without the domain or this will break depending on `.env` config.

Reply:
Same here as the `route()` discussion, I think we could use those helpers to make sure these checks are less hard-coded.